### ISSUE TITLE
ERROR Unexpected token ?﻿ in JSON at position 0

### ISSUE THREAD

**Issue by amrutha95** [GoogleChromeLabs/bubblewrap#679]
**Describe the bug**

The [issue](https://github.com/pwa-builder/PWABuilder/issues/2777) was raised on the [PWABuilder](https://pwabuilder.com/) repo. We use bubblewrap to package TWAs. 
<img width="316" alt="image" src="https://user-images.githubusercontent.com/22471775/165642587-1af95146-31a5-49b9-b4c4-1ee474c98da1.png">
The [manifest](https://tekye.net/manifest.json) of this PWA contains arabic letters which might be what's breaking the JSON parsing since we also have that issue while using node-fetch in one of our other APIs.

**To Reproduce**
bubblewrap init --manifest=https://tekye.net/manifest.json

**Expected behavior**
The JSON should be parsed with proper encoding.

**Comment by andreban** [comment]
The issue is that this manifest includes a [Byte Order Mark (BOM)](https://en.wikipedia.org/wiki/Byte_order_mark) section at the beginning of the file, [making it an invalid JSON](https://stackoverflow.com/questions/4990095/json-specification-and-usage-of-bom-charset-encoding). See data from an hex editor below:

![Screenshot 2022-04-28 at 14 50 16](https://user-images.githubusercontent.com/1733592/165768083-f9fc72d3-454d-458f-afc1-b3048930fce8.png)


To double check, I copied the contents of the file to another URL, removing the BOM section, and things work fine.

```
bubblewrap init --manifest=https://maroon-cloudy-halibut.glitch.me/manifest.json
```

This is unrelated to the encoding of the file. We can implement a workaround to detect/skip a BOM section when included in the file, but a faster workaround would be to remove the BOM section from the JSON file (and make it a properly valid JSON).

**Comment by andreban** [comment]
#685 works around this

**Comment by andreban** [comment]
Fixed via #685

**Comment by amrutha95** [comment]
I'm still running into this on the cli v1.17.0. Which version should I be on?
